test/controllers/CanPayWithinSixMonthsControllerSpec.scala

```diff
@@ -111,15 +111,35 @@ class CanPayWithinSixMonthsControllerSpec extends ItSpec with PegaRecreateSessio
         JourneyJsonTemplates.`Obtained Can Pay Within 6 months - no`(Origins.Epaye.Bta)(testCrypto)
       )
 
-      val result = controller.canPayWithinSixMonths(Epaye)(fakeRequest)
+      val result = controller.canPayWithinSixMonths(Epaye, None)(fakeRequest)
       testPageIsDisplayed(result, Some(false))
     }
 
+    "change the language cookie to english" in {
```

Comment:
it'd be worth having these sorts of tests for the other two endpoints that do the same thing as well I think so we've got full coverage